metadata/src/main/resources/common/metadata/PartitionChangeRecord.json

```diff
@@ -17,7 +17,9 @@
   "apiKey": 5,
   "type": "metadata",
   "name": "PartitionChangeRecord",
-  "validVersions": "0",
+  "validVersions": "0-1",
```

Comment:
Do we need to bump the version if we add just the tagged fields?

Reply:
Good question. Not super sure about this. @mumrah

Reply:
Skipped the version bump.